### ISSUE TITLE
Heat maps shrinks and model fails to detect text during fine tuning

### ISSUE THREAD

**Issue by MrAsifKhan** [backtime92/CRAFT-Reimplementation#107]
when I finetune the model on my dataset, the model performance is getting worse with the training. **The heatmaps starts to shrink and disappear after 10-20 epochs.**
Why is this happening and what is the solution for this??
 The loss seems to improve but the model fails to detect the text after few epochs.

appreciate the work @backtime92
